.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+Contributing to Conan
+*********************
+
+The following summarizes the process for contributing to the Conan project.
+
+
+Community
+=========
+
+Conan is an Open Source MIT licensed project. 
+
+Conan is maintained and supported by [JFrog](https://www.jfrog.com) and developed in collaboration with a great 
+community of contributors.
+
+
+Dev-flow & Pull Requests
+========================
+
+Conan follows the ["GitFlow"](https://datasift.github.io/gitflow/IntroducingGitFlow.html) branching model. 
+To contribute:
+
+1. Fork the [Conan main repository](https://github.com/conan-io/conan).
+2. Create a `feature/xxx` branch from the ``develop`` branch and develop your feature.
+3. Try to keep your branch updated with the ``develop`` branch to avoid conflicts.
+4. Open a pull request, and select ``develop`` as the base branch. Never open a pull request to ``master``
+ or ``release/xxx` branches.
+ 
+ 
+The ``conan-io`` organization users will prioritize the Issues and will assign them to a Milestone accordingly. 
+
+Each milestone corresponds to a Conan release, a branch ``release/xxx`` will be opened from the ``develop`` branch.
+When the branch is ready, it will be merged into ``master`` branch and a TAG corresponding to the version will be pushed.
+
+The ``conan-io`` organization users reserve the right of altering the Milestones and the issues of each Milestone,
+as well as the release dates.
+
+
+Issues
+======
+
+If you think you found a bug in Conan:
+
+- Explain the Conan version, Operating System, compiler and any other tool that could be related to the issue.
+- Explain, as detailed as possible, how to reproduce the issue.
+- Include what you expected to happen, as well as what actually happened.
+- If it helps, feel free to attach a screenshot or video illustrating the issue.
+
+For any suggestion, feature request or question:
```

Comment:
I would say that questions and support requests are very welcome. To make it clear. Also maybe it would be useful to use [question], [bug] tags in the title